app/navigation/houseConsignment/index/items/HouseConsignmentItemNavigator.scala

```diff
@@ -200,13 +211,58 @@ class HouseConsignmentItemNavigator(houseConsignmentMode: Mode) extends Navigato
             controllers.routes.HouseConsignmentController.onPageLoad(ua.id, houseConsignmentIndex)
         }
     }
+
+  private def netWeightPageRoute(ua: UserAnswers, houseConsignmentMode: Mode, itemMode: Mode, houseConsignmentIndex: Index, itemIndex: Index): Option[Call] =
+    (config.phase6Enabled, houseConsignmentMode, itemMode) match {
+      case (true, _, NormalMode) =>
+        Some(routes.UniqueConsignmentReferenceYesNoController.onPageLoad(ua.id, houseConsignmentIndex, itemIndex, houseConsignmentMode))
+      case (false, _, NormalMode) =>
+        Some(routes.AddCustomsUnionAndStatisticsCodeYesNoController.onPageLoad(ua.id, houseConsignmentIndex, itemIndex, houseConsignmentMode, itemMode))
+      case (_, CheckMode, CheckMode) => Some(controllers.routes.HouseConsignmentController.onPageLoad(ua.id, houseConsignmentIndex))
+    }
+
+  private def addUniqueConsignmentReferenceYesNoRoute(ua: UserAnswers, houseConsignmentIndex: Index, itemIndex: Index, itemMode: Mode): Option[Call] =
```

Comment:
We can assume that phase 6 is enabled. Hopefully removing it will mean we don't have these non-exhaustive pattern matches